docs/backup_and_restore.rst

```diff
@@ -31,13 +31,14 @@ You can use the following command to determine the volume of submissions
 currently on the *Application Server*: log in over SSH and run
 ``sudo du -sh /var/lib/securedrop/store``.
 
-.. note:: Submissions are deleted asynchronously and one at a time, so if you
-          delete a lot of submissions through the *Journalist Interface*, it may
-          take a while for all of the submissions to actually be
-          deleted. SecureDrop uses ``srm`` to securely erase files, which takes
-          significantly more time than normal file deletion. You can monitor the
-          progress of queued deletion jobs with ``sudo tail -f
-          /var/log/securedrop_worker/err.log``.
+.. note:: Submissions are deleted asynchronously and one at a time, so
+          if you delete a lot of submissions through the *Journalist
+          Interface*, it may take a while for all of the submissions
+          to actually be deleted. SecureDrop uses ``shred`` to
+          securely erase files, which takes significantly more time
+          than normal file deletion. You can monitor the progress of
+          queued deletion jobs with ``sudo tail -f
```

Comment:
would be good to explicitly mention "on the applcation server" and put the command in a code block for legibility